integration/aggregate_variables_compat_test.go

```diff
@@ -33,7 +33,7 @@ func TestAggregateVariablesCompatRoot(t *testing.T) {
 				bson.D{{"$addFields", bson.D{{"field", "$$ROOT"}}}},
 			},
 			failsForSQLite: "https://github.com/FerretDB/FerretDB/issues/3148",
-			skip:           "https://github.com/FerretDB/FerretDB/issues/1992",
+			skip:           "https://github.com/FerretDB/FerretDB/issues/1413",
```

Comment:
Actually `$set` and `$addFields` do not support expression yet. So let's not introduce that until they are implemented.